security/keycloak-authz/src/main/resources/application.properties

```diff
@@ -9,6 +9,9 @@ quarkus.oidc.token.lifespan-grace=60
 quarkus.keycloak.policy-enforcer.enable=true
 quarkus.keycloak.policy-enforcer.paths.health.path=/q/*
 quarkus.keycloak.policy-enforcer.paths.health.enforcement-mode=DISABLED
+// TODO: Workaround for https://github.com/quarkusio/quarkus/issues/17179
```

Comment:
Just curious: is this a legal comment syntax? I mean, it obvoiusly works, but still...